packages/calcite-components/src/components/input-date-picker/input-date-picker.e2e.ts

```diff
@@ -385,7 +385,6 @@ describe("calcite-input-date-picker", () => {
         await inputDatePicker.click();
         await page.waitForChanges();
         calendar = await page.find(`calcite-input-date-picker >>> .${CSS.calendarWrapper}`);
-
```

Comment:
@Elijbet This file still has a stylistic change.

Reply:
Sorry I'm still working on this.